sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/TracerProviderConfiguration.java

```diff
@@ -172,12 +172,14 @@ static Sampler configureSampler(
           return Sampler.parentBased(Sampler.traceIdRatioBased(ratio));
         }
       default:
```

Comment:
What do you think about adding a specific case for for `parentbased_jaeger_remote`? Something like:
```
      case "parentbased_jaeger_remote":
        Sampler jaeger_remote = spiSamplersManager.getByName("jaeger_remote");
        if (jaeger_remote == null) {
          throw new ConfigurationException("parentbased_jaeger_remote configured but opentelemetry-sdk-extension-jaeger-remote-sampler not on classpath");
        }
        return Sampler.parentBased(jaeger_remote);
```
This solution opens up the possibility of using any SPI sampler with the parentbased sampler, which is not part of the spec.

Reply:
The current implementation allows to load any sampler by name, even if it's not listed in the "Known values" of the spec. It's true that that spec doesn´t explicity say that a `parentbased_sampler` will  be a parent-based sampler of `sampler` but I find it hard to image a scenario where it isn't the case.
But if you don´t agree I'll change this PR to support only `parentbased_jaeger_remote` and leave the discussion for another time/PR.